conjure-java-core/src/main/java/com/palantir/conjure/java/ConjureTags.java

```diff
@@ -46,7 +46,7 @@ public static boolean isUnsafe(Collection<String> tags) {
 
     public static ImmutableList<AnnotationSpec> tagAnnotations(ArgumentDefinition argument) {
         validateTags(argument);
-        List<String> tags = argument.getTags();
```

Comment:
I think while using the local version of `conjure-api` I picked up another API change where tags became sets instead of lists. Guess this should be resolved as the other PRs resolve.